src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -12392,18 +12394,19 @@ private bool TryReadPlpUnicodeCharsChunk(char[] buff, int offst, int len, TdsPar
                 return true;
             }
 
-            charsRead = len;
+            int charsToRead = len;
 
             // stateObj._longlenleft is in bytes
-            if ((stateObj._longlenleft >> 1) < (ulong)len)
-                charsRead = (int)(stateObj._longlenleft >> 1);
+            if ((stateObj._longlenleft / 2) < (ulong)len)
```

Comment:
Why the change from '>> 1' to '/ 2'?

Reply:
I needed a petty change to force the CI to run. I think the division is clearer and that the jit will do the same thing but I'm not going to argue strongly either way.

Reply:
Bit shifting has always been confusing to me but after some "learning", I get it now. I have no problems with this either way.